share/server/util.js

```diff
@@ -138,6 +138,15 @@ function log(message) {
   // idea: query_server_config option for log level
   if (typeof message == "xml") {
     message = message.toXMLString();
+  } else if (typeof message === "object"
```

Comment:
worth DRYing this into a function to be reused in views.js?

Reply:
Good idea, thanks! Updated with a common function in utils.js